src/app/pages/ParaTimesPage/TransactionAmount/index.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+import { Box, Button, Form, FormField, Text, TextInput, Tip } from 'grommet'
+import { Trans, useTranslation } from 'react-i18next'
+import BigNumber from 'bignumber.js'
+import { formatBaseUnitsAsRose, isAmountGreaterThan } from 'app/lib/helpers'
+import { paraTimesActions } from 'app/state/paratimes'
+import { AlertBox } from 'app/components/AlertBox'
+import { defaultWithdrawFeeAmount } from 'app/lib/transaction'
+import { StringifiedBigInt } from 'types/StringifiedBigInt'
+import { AmountFormatter } from '../../../components/AmountFormatter'
+import { ParaTimeContent } from '../ParaTimeContent'
+import { ParaTimeFormFooter } from '../ParaTimeFormFooter'
+import { useParaTimes } from '../useParaTimes'
+import { useParaTimesNavigation } from '../useParaTimesNavigation'
+import { FeesSection } from './FeesSection'
+
+function getMaxAmountValue(
+  isDepositing: boolean,
+  accountBalance: StringifiedBigInt,
+  feeAmount: string,
+  balanceInBaseUnit: boolean,
+  consensusDecimals: number,
+  runtimeDecimals: number,
+): StringifiedBigInt {
+  const balance = balanceInBaseUnit
+    ? accountBalance
+    : new BigNumber(accountBalance).shiftedBy(consensusDecimals - runtimeDecimals).toFixed(0)
+  const maxValue =
+    isDepositing && !feeAmount
+      ? accountBalance
```

Comment:
Add comment: is this relying on `defaultDepositFeeAmount = '0'`, or are fees paid from different account's balance?

I assume the second, but then `&& !feeAmount` doesn't make sense